content/cockpit-137.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 136 and 137
+author: Dominik Perpeet
+date: 2017-03-30 22:00
+tags: cockpit, linux
+slug: cockpit-137
+category: release
+summary: Cockpit with menu and storage changes
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from versions 136 and 137.
+
+### Reordered entries in the sidebar menu
+
+As a followup to the new look, the entries in the sidebar menu have now been reordered in a more sensible fashion:
+
+ * System information
+ * Logs for troubleshooting
+ * Configuring major subsystems: Storage, Networking
+ * What's running: Containers and Virtual Machines
+ * Implementation details: Admin accounts, Services/Units
+
+Check out the screenshot below to see how this looks now.
+
+![Sidebar order](chttp://cockpit-project.org/blog/images/cockpit-menu-order.png)
```

Comment:
What's the magic behind these `chttp://` links? I first thought it was a typo, but it seems that all links in previous commit have that too.